### ISSUE TITLE
Add registration date field to WP Registration

### ISSUE THREAD

**Issue by davisagli** [washingtontrails/vms#5]
Matthew says "may even want multiple dates, for when status changes"

**Comment by mscholtz** [comment]
@ninetaLogik I think we need to add a date field to the object (make sure it's not already there).  Then set it to today when someone registers.  We can worry about changes to it later.

**Comment by davisagli** [comment]
It would probably be useful if it's a datetime, not just a date.

**Comment by 501Commons** [comment]
Added datetime field to WP Registration : Registration_Date__c. Modified the registration ws to set this field.

**Comment by ninetaLogik** [comment]
Added datetime field to WP Registration : Registration_Date__c. Modified the registration ws to set this field.

**Comment by davisagli** [comment]
I created a registration and was put on the waitlist, but the registration date was still set. It should only be set once the registration's status changes to Registered.

https://cs10.salesforce.com/a2CJ0000000UiZG

**Comment by ninetaLogik** [comment]
@mscholtz Setting the Registration Date would worked better in the trigger then , right?

**Comment by ninetaLogik** [comment]
@davisagli Sorry, I misunderstood what this fields was for. Please try again. It should work as expected now.

**Comment by davisagli** [comment]
I created another waitlisted registration from VMS, and the date did not get set, as expected.

However I then edited the registration in Salesforce and changed the status to Registered. I was expecting the date to get set at this point, but it was not. https://cs10.salesforce.com/a2CJ0000000Uibb

(We could probably wait and fix that when we work on story 40 which covers registering someone from the waitlist.)
